content-src/news_updates.yaml

```diff
@@ -8,25 +8,6 @@
 #   dev: false
 #   content: >
 #     Example content goes here
--
-  slug: general-update-3
```

Comment:
I'm not sure we should do this; there will be l10n conflicts in the future when we add new ones.